app/javascript/packs/application.js

```diff
@@ -21,3 +21,7 @@ import "bootstrap";
 import "../stylesheets/application";
 import "./custom";
 import "@fortawesome/fontawesome-free/js/all";
+// Support component names relative to this directory:
+var componentRequireContext = require.context("components", true);
+var ReactRailsUJS = require("react_ujs");
+ReactRailsUJS.useContext(componentRequireContext);
```

Comment:
move to separate file and require here

Reply:
and this code duplicates https://github.com/IF-115-Ruby/OSBB/pull/78/files#diff-c571eab111f6bc0e84fb6ad87b3425970e02a8a59dfcef1034a057fa4612d556R1